src/pages/People.js

```diff
@@ -0,0 +1,407 @@
+import AkaashPadmanabha from '../images/akaash_padmanabha.jpeg'
+import PEGLogo from '../images/peg.jpeg'
+import SSAPLogo from '../images/ssap.png'
+import PUCPLogo from '../images/pucp.jpeg'
+
+const Headshot = s.img`
+    width: 17vw;
+    height: 17vw;
+    object-fit: cover;
+    border-radius: 50%;
+    margin-bottom: 1vw;
+    margin-top: 3vw;
+`
+const Caption = s.p`
+    font-family: Roboto;
+    font-weight: bold;
+    font-size: 20px;
+    text-align: center;
+`
+
+//keynote
+const Keynote = () => {
+  return (
+    <div>
+      <Subtitle>Keynote Speakers</Subtitle>
+      <Grid columns={2} justifyContent="center">
+        <Cell center>
+          <Headshot src={ArunMajumdar} alt="Arun Majumdar" />
+        </Cell>
+        <Cell center>
+          <Headshot src={DennisWoodside} alt="Dennis Woodside" />
+        </Cell>
+
+        <Cell center>
+          <Caption>
+            <b>Dr. Arun Majumdar</b>
+            <br />
+            <Text css={`text-align:center`}>Founder and Director of ARPA-E, 
+              Former Head of Google Sustainability, Stanford Professor</Text>
+          </Caption>
+        </Cell>
+
+        <Cell center>
+          <Caption>
+              <b>Dennis Woodside</b>
+              <br />
+              <Text css={`text-align:center`}>President of Impossible Foods</Text>
+            </Caption>
+        </Cell>
+      </Grid>
+    </div>
+  )
+}
+//panelists
+const Panelists = () => {
+  return (
+    <div>
+      <Subtitle>Panelists</Subtitle>
+      <Grid columns={3} gap= "5px" justifyContent="center">
+        <Cell center>
+          <Headshot src={ZhengxiaDou} alt="Dr. Zhengxia Dou" />
+        </Cell>
+        <Cell center>
+          <Headshot src={DavdGalligan} alt="Dr. David T Galligan" />
+        </Cell>
+        <Cell center>
+          <Headshot src={JamesHagan} alt="Dr. James R Hagan" />
+        </Cell>
+        <Cell center>
+          <Caption>
+            <b>Dr. Zhengxia Dou</b>
+            <br />
+            <Text>Professor, Agricultural Systems, 
+              University of Pennsylvania
+              School of Veterinary Medicine, 
+              Department of Clinical Studies,
+              New Bolton Center</Text>
+          </Caption>
+        </Cell>
+
+        <Cell center>
+          <Caption>
+            <b>Dr. David T Galligan</b>
+            <br />
+            <Text>Professor of Animal Health
+              Economics, University of 
+              Pennsylvania, School of
+              Veterinary Medicine</Text>
+          </Caption>
+        </Cell>
+
+        <Cell>
+          <Caption>
+            <b>Dr. James R Hagan</b>
+            <br />
+            <Text>Graduate Student Advisor in the 
+              Sustainability and Environmental 
+              Policy concentrations, Former 
+              Vice President of Sustainability and Environment
+               at GlaxoSmithKline</Text>
+          </Caption>
+        </Cell>
```

Comment:
I didn't build this page, so idk but is there a reason this Cell isn't centered like the rest?